frontend/src/components/GVuelidateButton.vue

```diff
@@ -0,0 +1,84 @@
+<!--
```

Comment:
I have never seen or used a UI where error messages are shown as a tooltip for button. I would strongly recommend to solve the problem in a different way.